TCAT/Controllers/CustomNavigationController.swift

```diff
@@ -119,16 +167,30 @@ class CustomNavigationController: UINavigationController, UINavigationController
                 viewController.navigationItem.setLeftBarButton(customBackButton(), animated: true)
             }
         }
+        
+        if let homeMapVC = viewController as? HomeMapViewController {
+            reachabilityDelegate = homeMapVC
+        } else if let routeOptionsVC = viewController as? RouteOptionsViewController {
+            reachabilityDelegate = routeOptionsVC
+        }
```

Comment:
Do we want this to only work for certain viewControllers or is this simply a check to see if the viewController conforms to the protocol? If its the latter, then we can change this to be 
```swift
if let vc = viewController as? ReachabilityDelegate {
  ...
}
```